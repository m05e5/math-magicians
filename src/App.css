.App {
  text-align: center;
}

.App-body {
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
}

input[type="number"] {
  -moz-appearance: textfield;
}

input[type="number"]::-webkit-outer-spin-button,
input[type="number"]::-webkit-inner-spin-button {
  -webkit-appearance: none;
  margin: 0;
}

.calc {
  width: 500px;
}

.calc-input {
  font-family: 'Times New Roman', Times, serif;
  width: 99%;
  border: 1px solid #bdbdbd;
  height: 50px;
  background-color: grey;
  color: white;
  font-size: 25px;
  direction: rtl;
}

.calc-input:focus {
  outline: none;
}

.calc-buttons {
  width: 100%;
  font-family: arial, sans-serif;
  border-collapse: collapse;
}

td {
  border: 1px solid #bdbdbd;
  font-size: 18px;
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: center;
}

.bg-1 {
  background-color: rgb(223, 223, 223);
}

.bg-2 {
  background-color: rgb(236, 142, 54);
}
