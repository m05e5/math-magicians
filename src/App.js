/* eslint-disable no-useless-constructor */
/* eslint-disable react/prefer-stateless-function */
import './App.css';
import React from 'react';
import Calculator from './components/Calculator';

class App extends React.Component {
  constructor(props) {
    super(props);
  }

  /* eslint-enable */
  render() {
    return (
      <div className="App">
        <div className="App-body" id="App-body">
          <Calculator />
        </div>
      </div>
    );
  }
}

export default App;
