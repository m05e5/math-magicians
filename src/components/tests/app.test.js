import React from 'react';

import App from '../../App';

const ReactTestRenderer = require('react-test-renderer');

it('should render the Calculator App  component', () => {
  const tree = ReactTestRenderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
