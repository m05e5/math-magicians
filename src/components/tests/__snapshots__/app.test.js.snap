// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Calculator App  component 1`] = `
<div>
  <nav
    style={
      Object {
        "backgroundColor": "orange",
        "position": "fixed",
        "top": "0",
      }
    }
  >
    <h1>
      Math Magicians
    </h1>
    <div
      className="links"
    >
      <div
        className="link home_link"
      >
        <a
          href="/home"
          onClick={[Function]}
        >
          Home
        </a>
      </div>
      <div
        className="link cal_link"
      >
        <a
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </div>
      <div
        className="link"
      >
        <a
          href="/users"
          onClick={[Function]}
        >
          quote
        </a>
      </div>
    </div>
  </nav>
  <header>
    <h2>
      Welcome to our page!
    </h2>
    <p>
      A mathemagician is a mathematician who is also a magician. The term "mathemagic"  is believed to have been introduced by Royal Vale Heath with his 1933 book "Mathemagic".[1]
      <span
        className="name"
      >
        The name "mathemagician" was probably first applied to Martin Gardner, but has since been used to describe many mathematician/magicians, ncluding Arthur T. Benjamin,[2] Persi Diaconis,[3] and Colm Mulcahy.[4] Diaconis has suggested that the reason so many mathematicians are magicians is that "inventing a magic trick and inventing a theorem are very similar activities."[5]
      </span>
      <span
        className="name"
      >
        The Mathemagician is name of a character in the 1961 children's book The Phantom Tollbooth. He is the ruler of Digitopolis, the kingdom of mathematics.
      </span>
    </p>
  </header>
</div>
`;
